__tests__/container.test.js

```diff
@@ -0,0 +1,192 @@
+const { ContainerService } = require('../src/services')
+const { Container, FoodItem } = require('../src/models')
+const { initFirebase } = require('../src/firebase/config')
+
+//test to ensure the service can be created and used
+test('should pass health check without errors', async () => {
+    const services = setup()
+    const [containerService] = services
+
+    containerService.__HealthCheck('Message')
+})
+
+//CreateContainer
+test('should create container by params without errors', async () => {
+    const services = setup()
+    const [containerService] = services
+
+    const createdContainer = await containerService.createContainer('createByParams', 'someHouseholdId')
+    expect(createdContainer.id).toBeDefined()
+    expect(createdContainer.toDocument()).toMatchObject({
+        name: 'createByParams',
+        householdId: 'someHouseholdId',
+        foodItems: []
+    })
+})
+
+//GetContainer
+test('should get a container that has been created already without errors', async () => {
+    const services = setup()
+    const [containerService] = services
+
+    const containerCreated = await containerService.createContainer('getContainerTest1', 'someHouseholdId')
+    const retrievedContainer = await containerService.getContainerById(containerCreated.id);
+
+    expect(retrievedContainer).toMatchObject(containerCreated)
+    
+})
+
+test('should try to get a container that doesn\'t exist and return null without error', async () => {
+    const services = setup()
+    const [containerService] = services
+
+    const containerId = 'I_Do_Not_Exist'
+    const retrievedContainer = await containerService.getContainerById(containerId);
+
+    expect(retrievedContainer).toBe(null)
+
+})
+
+//DeleteContainer
+test('should create and delete a container by id without error', async () => {
+    const services = setup()
```

Comment:
Since you are not doing `teardown()` lines 52-53 can be combined into `const [serviceNameThatYouChoose] = setup()`